src/main/java/Task.java

```diff
@@ -0,0 +1,10 @@
+public class Task {
+    public int status;
```

Comment:
Should this be declared as non-public? I also see the same issue on some other variables.
Reference: "Class variables should never be declared public unless the class is a data class with no behavior."

Reply:
behaviour here means methods?

Reply:
I'm not 100% sure, but I think it means methods